addons/mail/static/src/components/message/message.xml

```diff
@@ -1,6 +1,20 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
 
+    <t t-name="mail.sendAgainNotification" owl="1">
```

Comment:
> I would define this below the main template, and check with @alexkuhn for name guideline in this case. (probably `mail.Message_sendAgain` or similar to class name?)

TBH don't really care as long as naming clearly maps with its structural location, so `mail.Message_sendAgain` or `mail.Message.sendAgain` are relatively good candidates.

_(wouldn't bother with guidelines now when my PR will revamp component templating. Basically, it will put much more emphasis on parts of a component template being clearly-and-uniquely named, and as self-explanatory as possible)_

Reply:
I don't like the fact that the name doesn't tell that the template is some sort of notification or error. But that's not really important here.

Reply:
It's good that you are questioning its name. I don't mind to rename it to something better, quite the opposite.

My comment was more in regard to general naming guideline of "template part" (so have a name that is the same as its class, and as Alex said, must keep the structural location in the name), not the name itself.